march_gazebo_plugins/CMakeLists.txt

```diff
@@ -8,10 +8,22 @@ find_package(catkin REQUIRED COMPONENTS
 )
 find_package(gazebo REQUIRED)
 
-catkin_package(CATKIN_DEPENDS march_shared_resources)
+catkin_package(
+    INCLUDE_DIRS include
+    CATKIN_DEPENDS march_shared_resources
+)
```

Comment:
`catkin_package` should contain the library you want to export
```suggestion
catkin_package(
    INCLUDE_DIRS include
    LIBRARIES ${PROJECT_NAME}
    CATKIN_DEPENDS march_shared_resources
)
```

Reply:
Thanks a lot this works :) Why is it necessary to put the header files inside a folder name ${PROJECT_NAME}?

Reply:
It's just convention and it might be necessary for installing